### PR TITLE
fix swagger api body bug

### PR DESCRIPTION
**Pull request by barshimi**

### CODE REVIEW

File: plugins/swagger-apibody/src/index.ts [hunk 1]
```diff
@@ -27,39 +31,45 @@ class SwaggerApiBody implements AmplicationPlugin {
...
-        )
-      );
+        classMethod?.decorators?.push(currDecorator);
+      });
 
-      classMethod?.decorators?.push(currDecorator);
-    });
+      return eventParams;
+    } catch (error) {
+      console.error("Failed to invoke beforeCreateControllerBase event under swagger-api-body plugin");
```
Comment (mulygottlieb) [inline plugins/swagger-apibody/src/index.ts:69]: Where will we be able to see this console.error when it's running inside the DSG container? is there a way to pass back the error to the build log?

Comment (overbit) [reply plugins/swagger-apibody/src/index.ts:69]: Let's use context.logger to get that message to the user. Plus we will also get it on our logging system.

Comment (barshimi) [reply plugins/swagger-apibody/src/index.ts]: good catch... I printed it to context.logger

File: plugins/swagger-apibody/src/index.ts [hunk 2]
```diff
@@ -27,39 +31,45 @@ class SwaggerApiBody implements AmplicationPlugin {
...
-        )
-      );
+        classMethod?.decorators?.push(currDecorator);
+      });
 
-      classMethod?.decorators?.push(currDecorator);
-    });
+      return eventParams;
+    } catch (error) {
+      context.logger.error("Failed to invoke beforeCreateControllerBase event under swagger-api-body plugin");
```
Comment (overbit) [inline plugins/swagger-apibody/src/index.ts:69]:
```suggestion
      context.logger.error("Failed to invoke beforeCreateControllerBase event under swagger-api-body plugin", error);
```

**Review by mulygottlieb — APPROVED** [review]

**Pull request merged**